### PR TITLE
chore(team-org-roles): drop team org_role field in django

### PR DESCRIPTION
**Pull request by cathteng**
The `org_role` column on the `Team` model is no longer being used. All code relating to it has been removed. This is part of an ongoing effort to remove the org roles for teams feature https://github.com/getsentry/team-core-product-foundations/issues/51

Part 1 of removing columns according to https://develop.sentry.dev/database-migrations/#deleting-columns

Only removes the column from the state (Django).

### CODE REVIEW

**Review by schew2381 — COMMENTED** [review]
lgtm from product side, will wait for migration owner to approve

**Review by markstory — APPROVED** [review]
Looks good.

**Review by wedamija — APPROVED** [review]

**Comment by azaslavsky** [comment]
This is the first time since this feature was introduced that we’ve run across someone deleting a field. I’m working on a solution, and should have something up for review in a few hours.

**Comment by azaslavsky** [comment]
I guess ~50 hours is technically still "a few". I've pushed two commits to this stack. The first is a general-case fix for schema field deletions. The second is the specific fix necessary for the field deletion done in this PR.

The first commit is a cherry-pick of a standalone PR at https://github.com/getsentry/sentry/pull/66248. Please wait for that to go in before merging this PR!

I've also made an issue, https://github.com/getsentry/sentry/issues/66247, for removing the shim, which will need to be completed once 24.4.0 is released in mid-April.

File: src/sentry/backup/imports.py
Comment (cathteng) [reply src/sentry/backup/imports.py:167]: ah i incorrectly rebased, thanks for catching!

**Pull request merged**
